Lib/test/decimaltestdata/abs.decTest

```diff
@@ -20,7 +20,7 @@
 version: 2.59
 
 -- This set of tests primarily tests the existence of the operator.
--- Additon, subtraction, rounding, and more overflows are tested
```

Comment:
The test data is from a third party, so please take this change out. It does not make sense to maintain our own version.

Reply:
all test data or just this file?

Reply:
I am sure just this file.